Source/WTF/wtf/SegmentedVector.h

```diff
@@ -222,14 +221,14 @@ namespace WTF {
     private:
         struct Segment {
             T entries[0];
+
+            std::span<T, SegmentSize> entriesSpan() { return unsafeMakeSpan<T, SegmentSize>(entries, SegmentSize); }
         };
```

Comment:
Seems like it would be better if the `entries` array was private. We should always access it through the span. Maybe even name them `m_entriesArray` and `entries`.